R/query.R

```diff
@@ -25,14 +25,14 @@ query <- function(language, source) {
 #' captures of the query for further use. Both functions return the same
 #' information, just structured differently depending on your use case. 
 #' 
-#' - `query_matches()` returns the captures first grouped by _predicate_, and
-#'   further grouped by _match_ within each predicate. This is useful if you
-#'   include multiple predicates in your query or if you expect multiple
+#' - `query_matches()` returns the captures first grouped by _pattern_, and
+#'   further grouped by _match_ within each pattern. This is useful if you
+#'   include multiple patterns in your query or if you expect multiple
 #'   captures per match.
```

Comment:
I'm less confident about this one, but I think you might mean "multiple matches per pattern"?